.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: go
 go:
-- 1.11.x
+- "1.11.x"
+- "1.12.x"
+- "1.13.x"
```

Comment:
I'm just curious: why three go versions?

Reply:
I didn't want to drop 1.11 if I didn't have to, even though it's no longer supported.

Then, I added in the supported versions :) In theory if it works on an older version it should work on the newer ones, but in practice you never know!

Reply:
and 1.11 didn't work, so out it goes!

Reply:
Makes sense! Thanks!